utils/util/src/main/scala/pl/touk/nussknacker/engine/util/exception/GenericRateMeterExceptionConsumer.scala

```diff
@@ -1,29 +1,17 @@
 package pl.touk.nussknacker.engine.util.exception
 
-import cats.data.NonEmptyList
 import pl.touk.nussknacker.engine.api.exception.{EspExceptionConsumer, EspExceptionInfo, NonTransientException}
-import pl.touk.nussknacker.engine.util.metrics.RateMeter
 
-trait GenericRateMeterExceptionConsumer extends EspExceptionConsumer {
+trait GenericRateMeterExceptionConsumer extends EspExceptionConsumer with ExceptionRateMeter {
```

Comment:
1. We count exceptions in both exception consumer and  `NodeCountingListener` ? is it ok? Won't be it doubled / shadowed? 
2. Can't it be composed instead of inherited? :-(

Reply:
I cleaned up a bit, left RateExceptionConsumer only in Flink with comment